pkl-doc/src/main/resources/org/pkl/doc/scripts/search-worker.js

```diff
@@ -177,12 +178,11 @@ function toWordStarts(characters) {
   return result;
 }
 
-const regexIsUppercase = /\p{Lu}/u
-
-const regexIsNumericCharacter = /\p{N}/u
```

Comment:
Declaring these outside of `toCharClass` was causing problems because `function toCharClass` gets hoisted to the top of the file, while these consts do not. So, the consts aren't actually available to the body of `function toCharClass`.

JS is silly sometimes..

Reply:
*Which* JS are you calling silly? :p